### PR TITLE
Add link to Google Providers Package System Tests Public Dashboard

### PR DESCRIPTION
**Pull request by fdemiane**
Add link to Google Providers Package System Tests Public Dashboard.

This dashboard shows the status of the system tests of the head revision of the Google Providers Package. The system tests are run using the head revision of Apache Airflow.

Those tests are running on Google's Infrastructure, and the results are shown in this [dashboard](https://storage.googleapis.com/providers-dashboard-html/dashboard.html).

### CODE REVIEW

**Comment by potiuk** [comment]
👀

**Comment by fdemiane** [comment]
@potiuk Still a draft, testing a few last things before making the link accessible

File: tests/system/providers/google/README.md [hunk 1]
```diff
@@ -97,3 +97,7 @@ Keep in mind that some additional commands may be required.
 
 Some tests may require extra setup. If this is the case, the steps should be documented inside the docstring of
 related test file.
+
+## Google Providers Package System Tests Dashboard
```
Comment (kosteev) [inline tests/system/providers/google/README.md]: Nit: as the top section of README is named "Google provider system tests", I would consider to name this nested section e.g. simply "Dashboard" to not duplicate words. But I do not have very strong opinion on this, this way it looks also good

**Review by kosteev — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Shall I merge :) ?

**Comment by fdemiane** [comment]
@potiuk Yes please :) I just changed the link to deal with some accessibility

**Comment by vincbeck** [comment]
Nice!

**Pull request merged**